lib/dotcom_web/controllers/schedule/line_api.ex

```diff
@@ -90,134 +77,4 @@ defmodule DotcomWeb.ScheduleController.LineApi do
   end
 
   defp other_route_stops(_), do: []
-
-  @doc """
-  Provides predictions and vehicle information for a given route and direction, organized by stop.
-  The line diagram polls this endpoint for its real-time data.
-  """
-  @spec realtime(Plug.Conn.t(), map()) :: Plug.Conn.t()
-  def realtime(conn, %{"id" => route_id, "direction_id" => direction_id}) do
-    case LineHelpers.get_route(route_id) do
-      {:ok, route} ->
-        conn =
-          conn
-          |> assign(:route, route)
-          |> assign(:direction_id, String.to_integer(direction_id))
-          |> assign_vehicle_tooltips([])
-
-        payload =
-          do_realtime(
-            route_id,
-            direction_id,
-            conn.assigns.date,
-            conn.assigns.date_time,
-            conn.assigns.vehicle_tooltips
-          )
-
-        conn
-        |> put_resp_content_type("application/json")
-        |> send_resp(200, payload)
-
-      :not_found ->
-        return_invalid_arguments_error(conn)
-    end
-  end
-
-  def realtime(conn, _params) do
-    return_invalid_arguments_error(conn)
-  end
-
-  defp do_realtime(route_id, direction_id, date, now, tooltips) do
-    headsigns_by_stop =
-      TransitNearMe.time_data_for_route_by_stop(
-        route_id,
-        String.to_integer(direction_id),
-        date: date,
-        now: now
-      )
```

Comment:
I believe this was the only place this function was used, so you can kill it too :)